context/src/main/resources/ws/README.md

```diff
@@ -1,4 +1,4 @@
 # IMPORTANT
 
 DO NOT MODIFY THE FILES IN THIS FOLDER DIRECTLY. THE EXPECTATION IS
```

Comment:
Not sure if we want to add a section on testing. Hemanta asked me to user darpc_midtier_user for testing

Reply:
Perhaps, I can try to add a section here if that's useful.
In my case, I used SoapUI to manually hit the endpoints on http://localhost:8070/service/darts
with the ServiceContext Header with `RepositoryIdentity` similar to [Token Service](https://tools.hmcts.net/confluence/display/DMP/Token+Service) - see UserName/Password.
But we don't have that user currently - so I simply used what we had already in our https://github.com/hmcts/darts-gateway/blob/master/src/main/resources/application.yaml#L35 i.e. FUNC_TEST_ROPC_USERNAME

Reply:
Done in commit 08c8c3f 
* Update README: DARTSService SoapUI - ServiceContext Soap Header examples
* Add README-DARTSService-soapui-project.xml

See https://github.com/hmcts/darts-gateway/blob/DMP-1638-authentication/README.md